modules/ingest-common/build.gradle

```diff
@@ -18,8 +18,8 @@
  */
 
 esplugin {
-    description 'Ingest processor that uses grok patterns to split text'
-    classname 'org.elasticsearch.ingest.grok.IngestGrokPlugin'
+    description 'Module for ingest processors that do not require additional security permissions.'
```

Comment:
Good point. I'll adjust the description, so that it also describes that we don't want processors with large resources / dependencies in this module.